maat-court-data-api/src/main/java/gov/uk/courtdata/repository/ContribAppealRulesRepository.java

```diff
@@ -0,0 +1,18 @@
+package gov.uk.courtdata.repository;
+
+import gov.uk.courtdata.entity.ContribAppealRulesEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.math.BigDecimal;
+
+@Repository
+public interface ContribAppealRulesRepository extends JpaRepository<ContribAppealRulesEntity, Integer> {
+
+    @Query(value = "SELECT contrib_amount FROM TOGDATA.CONTRIB_APPEAL_RULES car WHERE car.CATY_CASE_TYPE = :caseType " +
```

Comment:
Can we move away from using native query please?

Reply:
replaced native query with projection